R/github.R

```diff
@@ -0,0 +1,39 @@
+#' See if a name is available on github
+#'
+#' @param name Name of package to search
+#' @importFrom jsonlite fromJSON
+#' @export
+available_github <- function(name) {
+  github_names <- github_packages()
+
+  same <- tolower(name) == tolower(github_names)
+  if (any(same)) {
+    stop(sprintf(
+      "`%s` conflicts with %s",
+      name,
+      paste0("`", github_names[same], "`", collapse = ", ")
+    ), call. = FALSE)
```

Comment:
Lets change this to return data rather than messages / errors

```r
tibble::tibble(source = "github", available = same, close = list(close_names))`
```

Where `close_names` is a character vector of existing names with X edit distance of `name`.